code/mission/missionparse.cpp

```diff
@@ -421,6 +421,9 @@ const std::shared_ptr<scripting::Hook> OnDepartureStartedHook = scripting::Hook:
 		{"Ship", "ship", "The ship that has begun the departure process."},
 	});
 
+ const std::shared_ptr<scripting::Hook> OnLoadoutParseHook = scripting::Hook::Factory("On Loadout Parse",
```

Comment:
After consultation with @BMagnu, I recommend calling this "On Loadout About to Parse" to make it clear that parsing has not actually happened yet.  Most "On X" hooks are fired after X has happened.

Reply:
"On Loadout Before Parse"?